model/simulationtests/tablelookup.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+require 'openstudio'
+require_relative 'lib/baseline_model'
+
+m = BaselineModel.new
+
+# make a 1 story, 100m X 50m, 1 zone core/perimeter building
+m.add_geometry({ 'length' => 100,
+                 'width' => 50,
+                 'num_floors' => 1,
+                 'floor_to_floor_height' => 3,
+                 'plenum_height' => 1,
+                 'perimeter_zone_depth' => 0 })
+
+# add windows at a 40% window-to-wall ratio
+m.add_windows({ 'wwr' => 0.4,
+                'offset' => 1,
+                'application_type' => 'Above Floor' })
+
+# Add ASHRAE System type 02, PTHP
+m.add_hvac({ 'ashrae_sys_num' => '02' })
+
+# add thermostats
+m.add_thermostats({ 'heating_setpoint' => 24,
+                    'cooling_setpoint' => 28 })
+
+# assign constructions from a local library to the walls/windows/etc. in the model
+m.set_constructions
+
+# set whole building space type; simplified 90.1-2004 Large Office Whole Building
+m.set_space_type
+
+# add design days to the model (Chicago)
+m.add_design_days
+
+###############################################################################
+#         R E P L A C E    A T Us    W/    F O U R    P I P E    B E A Ms
+###############################################################################
```

Comment:
Why write this?

Reply:
That's definitely a dumb copy paste.

Reply:
that's definitely a dumb copy paste error.

Reply:
Fixed in ff114b8694a2b1be3205254432a9741ec98892a0, thanks for catching